examples/diff_adv/burger.jl

```diff
@@ -18,6 +18,8 @@ using MLStyle
 using ComponentArrays
 using OrdinaryDiffEq
 
+## Original implementation can be found at: https://github.com/AlgebraicJulia/DECAPODES-Benchmarks
```

Comment:
Comment on wrong file

Reply:
That's true, now only CHT and CHT_cuda have this link.